### PR TITLE
added agreement lot page before start page.

### PR DESCRIPTION
**Pull request by ulysses-cognizant**
### JIRA link
https://crowncommercialservice.atlassian.net/browse/SCAT-914

### Change description
Create the before start page for agreement lot.

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Check and send page updated
- [ ] UI changes look good on mobile

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by prathap-cognizant — APPROVED** [review]

File: src/main/features/agreement/views/lot.njk [hunk 1]
```diff
@@ -6,30 +6,16 @@
...
-    
-  
-    
     <li class="govuk-breadcrumbs__list-item">
       <a class="govuk-breadcrumbs__link" href="/agreement?id=RM6263">Digital Specialists and Programmes</a>
     </li>
-    
-  
-    
-    <li class="govuk-breadcrumbs__list-item" aria-current="page">Lot 1: Programmes </li>
```
Comment (prathap-cognizant) [inline src/main/features/agreement/views/lot.njk:23]: need to make the as parametrise something like below. Don't hard code page title as Lot 1 it's should be dynamic based on query parameter.

`{{agreement_lot.number}} Programmes `

File: src/main/features/agreement/views/lot.njk [hunk 2]
```diff
@@ -41,45 +27,38 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full govuk-grid-column-two-thirds-from-desktop">
         <div class="project-overview">
-            <p class="govuk-body">Agreement name: <strong>Digital Specialists and Programmes</strong></p>
-            <p class="govuk-body">Agreement ID: <strong>RM6263</strong></p>
-            <p class="govuk-body">Expiry date: <strong>DD month YYYY</strong></p>
+            <p class="govuk-body">Agreement name: <strong>{{agreement_lot.name}}</strong></p>
```
Comment (prathap-cognizant) [inline src/main/features/agreement/views/lot.njk]: This is not a lot name, it's agreement name

File: src/main/features/agreement/views/lot.njk [hunk 3]
```diff
@@ -41,45 +27,38 @@
...
+            <p class="govuk-body">Agreement ID: <strong>{{agreement_id}}</strong></p>
+            <p class="govuk-body">Expiry date: <strong>{{agreement_lot.endDate}}</strong></p>
```
Comment (prathap-cognizant) [inline src/main/features/agreement/views/lot.njk]: This is agreement expiry date not the lot expiry date

**Review by prathap-cognizant — CHANGES_REQUESTED** [review]
please address all the review comments

**Review by viswa-cognizant — APPROVED** [review]

**Review by viswa-cognizant — APPROVED** [review]

File: src/main/features/agreement/views/lot.njk (hunk 3 above)
Comment (ulysses-cognizant) [reply src/main/features/agreement/views/lot.njk:32]: I took it form her
lotname

Comment (ulysses-cognizant) [reply src/main/features/agreement/views/lot.njk]: You mean, it should be this?
agreementENDDATE

File: src/main/features/agreement/views/lot.njk (hunk 2 above)
Comment (ulysses-cognizant) [reply src/main/features/agreement/views/lot.njk]: You mean, it should be this:
agreementNAME

File: src/main/features/agreement/views/lot.njk (hunk 1 above)
Comment (ulysses-cognizant) [reply src/main/features/agreement/views/lot.njk:23]: This is what is being deleted, right below you can find that it is dynamic.

**Review by CesarPerez — APPROVED** [review]

**Review by daimdev — APPROVED** [review]

**Review by prathap-cognizant — APPROVED** [review]

**Pull request merged**